core/src/main/scala/org/apache/spark/sql/delta/DeltaConfig.scala

```diff
@@ -394,6 +394,16 @@ trait DeltaConfigsBase extends DeltaLogging {
     _ => true,
     "needs to be a boolean.")
 
+  val EXTERNAL_POST_COMMIT_HOOK_CLASS = buildConfig[Option[String]](
+    "postCommitHookClass",
+    null,
+    v => Option(v),
+    a => a.nonEmpty,
+    "This class must be a subclass of PostCommitHook, " +
+      "and must have a zero-arg constructor. " +
+      "Note: This configuration will not changed for next transactions, " +
+      "once it is added to delta table, need to run alter command to remove or chance this.")
```

Comment:
"...to a delta table, ALTER TABLE is needed to drop or change it"

Reply:
fixed